infra/aws/terraform/prow-build-cluster/README.md

```diff
@@ -243,3 +243,31 @@ make destroy
 
 If you want to remove roles used for EKS creation go to `../iam/<aws_account_name>` and run `terraform destroy` command there.
 
+
+## GitOps
```

Comment:
I think we should also document:

- How to access Flux?
- How to monitor Flux?
- How to add resources to be deployed by Flux?
- How to watch for progress, e.g. if deployment is successful?